htdocs/expedition/card.php

```diff
@@ -1599,7 +1599,7 @@
 						$srcLine = new OrderLine($db);
 						$srcLine->id = $line->id;
 						$srcLine->fetch_optionals(); // fetch extrafields also available in orderline
-						$line->array_options = array_merge($line->array_options, $srcLine->array_options);
+						$expLine->array_options = array_merge($line->array_options, $srcLine->array_options);
```

Comment:
Having expLine + line + srcLine in same line looks strange.
Don't you want to make 
$expLine->array_options = array_merge($expLine->array_options, $srcLine->array_options);
instead of
$expLine->array_options = array_merge($line->array_options, $srcLine->array_options);

Reply:
You are correct, $line is orderline